lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    @cells = [' '] * 9
+  end
+
+  def initialize
+    reset!
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def input_to_index(user_input)
+    user_input.to_i - 1
+  end
+
+  def position(user_input)
+    user_index = input_to_index(user_input)
```

Comment:
Is there a way we can make this adjustment one time so that we don't have to call `#input_to_index` at the beginning of every method?